Tests/mock_server.py

```diff
@@ -194,12 +195,18 @@ def __init__(self, public_ip,
         silence_output(self.ami.call, ['mkdir', '-p', tmp_folder], stderr='null')
 
     @staticmethod
-    def configure_proxy_in_demisto(client, proxy=''):
-        system_conf_response = demisto_client.generic_request_func(
-            self=client,
-            path='/system/config',
-            method='GET'
-        )
+    def configure_proxy_in_demisto(username, password, server, prints_manager, thread_index, proxy=''):
+        client = demisto_client.configure(base_url=server, username=username,
+                                          password=password, verify_ssl=False)
+        try:
```

Comment:
no need for the try-except, in case this fails we want to stop.
otherwise, the rest of the tests will run without proxy

Reply:
I need to catch the error to prevent the headers from displaying. Would executing a `sys.exit()` work?

Reply:
we are catching the exception at a higher level, handle it there